picard/const/defaults.py

```diff
@@ -101,24 +102,24 @@
 
 DEFAULT_THEME_NAME = str(UiTheme.DEFAULT)
 
-DEFAULT_TOOLBAR_LAYOUT = [
-    'add_directory_action',
-    'add_files_action',
-    'separator',
-    'cluster_action',
-    'separator',
-    'autotag_action',
-    'analyze_action',
-    'browser_lookup_action',
-    'separator',
-    'save_action',
-    'view_info_action',
-    'remove_action',
-    'separator',
-    'cd_lookup_action',
-    'separator',
-    'submit_acoustid_action',
-]
+DEFAULT_TOOLBAR_LAYOUT = (
+    MainAction.ADD_DIRECTORY,
+    MainAction.ADD_FILES,
+    '-',
```

Comment:
We could, but I kept support for both, and, on write, it will use '-' from now.
Since that's quite simple, I don't think we need an upgrade hook (or we can do next time we add an upgrade hook).

Reply:
I see, that works